src/clr-angular/icon/icon.module.ts

```diff
@@ -5,13 +5,27 @@
  */
 
 import {CommonModule} from "@angular/common";
-import {NgModule, Type} from "@angular/core";
+import {isDevMode, NgModule, Type} from "@angular/core";
+
 import {ClrIconCustomTag} from "./icon";
 
 export const CLR_ICON_DIRECTIVES: Type<any>[] = [ClrIconCustomTag];
 
+// TODO: remove this warning in v0.13
+let warningThrownOnce = false;
+const DEPRECATED_SHAPE_WARNING =
+    `Starting in Clarity Icons v0.12, some icons are deprecated in their respective sets and moved to other sets. \n` +
```

Comment:
No need to use concatenation with multiline string, nor the `\n` character, just make a newline.

I'd do something a little more like first and last lines being something like:

```
*********
```